### DISCUSSION TITLE
Set Startboundary for multiple trigger types in the Wizard GUI?

### DISCUSSION THREAD

**Discussion by matthew-mayer** [dahall/TaskScheduler#914] [Q&A] [answered]
Hello, I'm in the process of developing an app that utilizing a customized basic wizard gui. Is there a way to set the start boundary time for multiple trigger types (time, daily, monthlyDOW, weekly) and have that represented in the wizard? I would like to set the same start time for all trigger types if possible.

Thanks!

**Comment by dahall** [comment, marked as answer]
Are you using [`TaskSchedulerWizard`](https://dahall.github.io/TaskScheduler/html/T_Microsoft_Win32_TaskScheduler_TaskSchedulerWizard.htm) or one you've created? The design of `TaskSchedulerWizard` doesn't really lend itself to handling multiple triggers.
